tests/input/src/main/scala/semantic/Println.scala

```diff
@@ -0,0 +1,18 @@
+/*
+mode = blacklist
+rules = [
+  NoPrintln
+]
+*/
+
+package res
+
+class Println {
+  
+  def prtl(x: String): Unit = println(x) /*
+                              ^^^^^^^^^^
+  println is not functional */
+
+  def prtls(xs: List[String]): Unit = xs.foreach(println) // has no span
```

Comment:
Well, the method is applied after eta expansion and is caught but has no span.

Reply:
So, this means that errors with no spans can be indicated in a test specification, right? That’s an issue, I think.

Reply:
Yes. Basically we need to avoid such `NoSpan` cases. After eta expansion, the `APPLY` tree has no span but the `fn_Term` tree (in this case `println`) does have a span. The quick fix is if the span of `APPLY` is missing, fall back to the span of `fn_Term`.

Reply:
It won't work because `NoSpan` is different from a zero-length span (or `Coord`). `NoSpan` doesn't have any positional information.

Reply:
When fixing this problem, I found another bug of the spans and started a [pull request](https://github.com/scalacenter/tasty-query/pull/66).